api/layer/neofs_mock.go

```diff
@@ -194,6 +270,32 @@ func (t *TestNeoFS) CreateObject(_ context.Context, prm PrmObjectCreate) (oid.ID
 	obj.SetOwnerID(&prm.Creator)
 	t.currentEpoch++
 
+	if prm.Multipart != nil && prm.Multipart.SplitID != "" {
+		var split object.SplitID
+		if err := split.Parse(prm.Multipart.SplitID); err != nil {
+			return oid.ID{}, fmt.Errorf("split parse: %w", err)
+		}
+		obj.SetSplitID(&split)
+
+		if prm.Multipart.SplitPreviousID != nil {
+			obj.SetPreviousID(*prm.Multipart.SplitPreviousID)
+		}
+
+		if len(prm.Multipart.Children) > 0 {
+			obj.SetChildren(prm.Multipart.Children...)
```

Comment:
@cthulhu-rider, @roman-khimov, @smallhive, just to be sure: we are not implementing v2 object split yet, right?

Reply:
Yes, it's to be implemented in the next version. We need to properly test the current scheme and then adapt it to v2.